app/src/controllers/dashboard/selectors.js

```diff
@@ -40,4 +41,6 @@ export const totalDashboardsSelector = (state) =>
 
 export const dashboardFullScreenModeSelector = (state) => domainSelector(state).fullScreenMode;
 
-export const querySelector = createQueryParametersSelector();
+export const querySelector = createQueryParametersSelector({ namespace: NAMESPACE });
```

Comment:
Do we need a `namespace` here?
In general the `namespace`s are used when there are several types of query params for different entities (like we can have several test item params & log params) simultaneously. In this case (like the similar one for filters, members, etc.) the only one set of query params possible - only dashboards. So, let's remove the namespace usage for query params if its redundant.